src/headless-reducer-manager.ts

```diff
@@ -17,12 +17,29 @@ import createSearchStatusSlice from './slices/searchstatus';
  * Manages the current map of headless IDs to Reducers.
  */
 export default class HeadlessReducerManager {
+  /**
+   * A mapping of unique headless IDs to Reducers.
+   */
   private headlessIdToReducer: Record<string, Reducer> = {};
 
+  /**
+   * Adds a mapping of the provided headless ID to an Answers state Reducer into
+   * {@link headlessIdToReducer}.
+   *
+   * @param headlessId - The ID of the AnswersHeadless instance for which to add a
+   *                     Reducer
+   */
   addAnswersReducer(headlessId: string): void {
     this.headlessIdToReducer[headlessId] = createAnswersReducer(headlessId + '/');
   }
 
+  /**
+   * Gets the parent Reducer for the {@link ParentState} of the mapping of
+   * AnswersHeadless instances.
+   *
+   * @returns A parent Reducer function that is a combination of the Reducers in
+   *          {@link headlessIdToReducer}
```

Comment:
I'm not sure that this link will work. I would expect that it would be {@link HeadlessReducerManager.headlessIdToReducer}. We could figure that out once you add API extractor, however since this class is internal, I don't think it will process it

Reply:
Removed this since it's internal